sdk/openai/openai/test/utils/logger.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Should we add logging file back for the browser test?

Reply:
If you're not doing it in your PR, I can add it in a follow-up PR.

Reply:
I just added it in https://github.com/Azure/azure-sdk-for-js/pull/32920/commits/49186fe0f16995b7380995862346a2a57860279c